README.md

```diff
@@ -68,13 +68,15 @@ The following command line arguments can be passed to all commands:
 -   `--io-package`: Path to the io-package.json file. Short: `-p`. Default: `./io-package.json`
 -   `--admin`: Path to the admin directory. Short: `-a`. Default: `./admin`
 -   `--words`: Path to the words.js file. Short: `-w`. Default: searches it in the admin directory, either `<admin-dir>/words.js` or `<admin-dir>/js/words.js`.
--   `--base`: Path to the english i18n file, multiple files are possible. Short: `-b`. Default: searches it in the admin directory, it will be `<admin-dir>/i18n/en/translations.json` or/and `<admin-dir>/src/i18n/en.json`
+-   `--base`: Path to the english i18n file, multiple files are possible. Short: `-b`. Default: searches it in the admin directory, it will be `<admin-dir>/i18n/en/translations.json` or/and `<admin-dir>/src/i18n/en.json`: Specify a subset of languages to be translated. Short `-l`. Default: all languages.
+-   `--languages:
```

Comment:
Please complete the documentation. I think something was lost here (not even the code "escape" is completed properly